From 2da5f5f58385d0c836fac627110bb9f1ac1c97ec Mon Sep 17 00:00:00 2001
From: Sughosh Ganu <sughosh.ganu@linaro.org>
Date: Thu, 9 Jan 2020 19:39:45 +0530
Subject: [PATCH 1/1] MAINTAINERS: Add entry for rng drivers

Take up maintainership of random number generator drivers with
Heinrich Schuchardt as the reviewer.

Signed-off-by: Sughosh Ganu <sughosh.ganu@linaro.org>
Reviewed-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 MAINTAINERS | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index 438fb225ab..a0f1eaf3fd 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -781,6 +781,15 @@ F:	arch/riscv/
 F:	cmd/riscv/
 F:	tools/prelink-riscv.c
 
+RNG
+M:	Sughosh Ganu <sughosh.ganu@linaro.org>
+R:	Heinrich Schuchardt <xypron.glpk@gmx.de>
+S:	Maintained
+F:	drivers/rng/
+F:	drivers/virtio/virtio_rng.c
+F:	cmd/rng.c
+F:	include/rng.h
+
 ROCKUSB
 M:	Eddie Cai <eddie.cai.linux@gmail.com>
 S:	Maintained
-- 
2.24.1

